plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/guru/VMwareGuru.java

```diff
@@ -1104,13 +1115,18 @@ public List<Command> finalizeMigrate(VirtualMachine vm, Map<Volume, StoragePool>
             if (pool.getClusterId() != null) {
                 poolClusterId = pool.getClusterId();
             }
+            if (volume.getVolumeType().equals(Volume.Type.ROOT) && pool.isLocal()) {
```

Comment:
```suggestion
            if (volume.getVolumeType().equals(Volume.Type.ROOT)) {
```

Reply:
Removed it from another method where we are getting target host guid. Please review @sureshanaparti